RawVideo/UIHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+
+namespace RawVideo
+{
+    public static class UIHelper
+    {
+        public static Task RunOnUIThread(this Page p, Action a)
+        {
+#if WINDOWS_UWP
```

Comment:
Do we need this ifdef here? This is a UWP to start with.

Reply:
done